airflow/models/taskinstance.py

```diff
@@ -885,6 +887,12 @@ def _run_raw_task(
                 context = self.get_template_context()
 
                 task_copy = copy.copy(task)
+
```

Comment:
It would be great to explain why in a comment.

Reply:
Done